app/static/js/main.js

```
@@ -0,0 +1,77 @@
+
+    // Swipe gestures for mobile devices
+    let touchstartX = 0;
+    let touchendX = 0;
+    const tabsElement = document.getElementById('weekdaysTabs');
+
+    function handleGesture() {
+        if (touchendX < touchstartX) {
+        const activeTab = document.querySelector('.nav-link.active');
+        if (activeTab) {
+            const nextTab = activeTab.parentElement.nextElementSibling;
+            if (nextTab && nextTab.querySelector('.nav-link')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)